resources/pod.go

```diff
@@ -0,0 +1,5 @@
+package resources
+
+// Pod is used to relate to Kubernetes pods
+type Pod struct {
+}
```

Comment:
You can add `Name` to the pod, as that's the only data we'll be tracking initially